source/dev/idea_list.md

```diff
@@ -0,0 +1,165 @@
+    - Download external data sources
+    - Validate and clean the downloaded resources
+    - Archive downloaded resources to a blob store
+- Workflows are instrumented with telemetry
+- Telemetry are aggregated in InfluxDB
+- Metrics are displayed in a Grafana dashboard
+- If a data source is down, we throw an alarm
+
+The workflows should be well documented with a focus on code quality, and use best-of-breed current technology stacks (Apache Airflow, Kubernetes, Helm, Docker, Python3, Grafana, etc.)
+
+**Requirements:**
+Docker, Python, Grafana, PAAS, Apache Airflow
+
+**Level:**
+Medium to Hard
+
+**Mentor:**
+[Todd](https://github.com/ToddG)
+[Jon](https://github.com/jon-hagg)
+
+### Project 2. - Test system for end-to-end testing of the simulation framework
+Use [existing power systems test grids](http://labs.ece.uw.edu/pstca/) in order to to benchmark our simulation framework
+Generate input data to feed to the simulation engine using data science techniques.
+Write integration tests and produce documentation to showcase our open source software.
+
+- Collect data (python scripting)
+- Generate time series for demand, hydro, solar and wind matching the test grid (python data analysis)
+- Integrate the test model in our [simulation framework](https://github.com/Breakthrough-Energy/PowerSimData) (python)
+- Write end-to-end tests (in docker container)
+- Write GitHub actions to test changes of the framework when pushed on GitHub
+- Write documentation for the community (sphinx)
+
+**Requirements:**
+Object Oriented Programming in Python, Data analysis/plotting in Python (pandas, numpy, matplotlib, etc.) and docker
+
+**Level:**
+Easy
+
+**Mentor:**
+[Ben](https://github.com/rouille)
+[Todd](https://github.com/ToddG)
+
+### Project 3. - Time-Series Modeling of Missing Energy Industry Data and Weather Data
+A significant barrier to realistic modeling of the power grid is missing data from public data sources.
+This project is to use statistical and ML models to fill in missing energy demand and renewable energy profile information in a way which mimics the real data as close as possible.
+In our current data sets of regional energy demand, we have used averaging of adjacent days to fill in our missing data, as described [here](https://github.com/Breakthrough-Energy/PreREISE#d-demand-data)
+
+We propose the following steps for this project:
+
+- Understand the data set and the [current methodology](https://github.com/Breakthrough-Energy/PreREISE#d-demand-data)
+- Literature review to understand prediction models for time series data
+- Collect data to build training/testing sets
+- Evaluate algorithms using cross validation or other metrics
+- Integrate most suitable algorithm(s) in our code base and generate new time series for our power system model
+
+**Requirements:**
+Python, Pandas, understanding of statistics and ML concepts
+
+**Level:**
+Medium
+
+**Mentor:**
+[Daniel Muldrew](https://github.com/dmuldrew)
```

Comment:
Daniel Muldrew --> Daniel since we use first name only for other mentors

Reply:
I want to make sure students can find us on Slack and there are more than one Daniel.